tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
+          position: absolute;
+          margin-left: 26px;
```

Comment:
using left instead of margin-left here would be preferable. Did you check if the new button and the div that appears when 'show download links' is enabled overlap?